docs/glossary.rst

```diff
@@ -0,0 +1,66 @@
+===============================================================================
+XBRL Glossary
+===============================================================================
+
+Axis
+^^^^
+
+Axes are defined in the Taxonomy, and are used as a part of the
+Context to identify a Fact beyond the default
+entity, and Period. Axes can be either typed or explicit. An
+explicit Axis will have all of its allowable values defined in the taxonomy. A
+typed Axis will instead allow a filer to input their own values for the Axis.
+
+Concept
```

Comment:
What does an XBRL Concept correspond to in the extracted relational database structure? Is it a data type? Is it a variable that's being reported across many filings in a single table? Can the same Concept appear in multiple tables, and if it does will it always have the same meaning?

Reply:
The relationship is a bit hard to describe, because _some_ Concepts will become columns, but not all of them. I think the figure in the parsing section hopefully illustrates this better. I also added a note about data types, because Concepts have a data type associated with them, that are used to deduce the data types used in the database